modules/ingress/variables.tf

```diff
@@ -15,15 +15,17 @@ variable "annotations" {
 
 variable "path" {
   type = list(object({
-    service_name = string
-    service_port = string
-    path         = string
+    service_name        = string
+    service_port_number = string
+    service_port_name   = string
+    path                = string
   }))
   default = [
     {
-      service_name = "dummy"
-      service_port = "80"
-      path         = "/alb-terraform-created"
+      service_name        = "response-200"
```

Comment:
I suggest we hardcode this in the module as user does not need to know about limitations and should only supply own variables.